### ISSUE TITLE
Execute the target?

### ISSUE THREAD

**Issue by ci70** [jbaltie/MSBuildSelector#2]
Is this plugin able to execute the target after its built?

I don't see a Run or a Clean option. Only build.

**Comment by ghost** [comment]
Not for now, the `Clean` seems interesting, I am less convinced by the `Run`.

Good idea, will try to implement this.

**Comment by ci70** [comment]
How else can you run the target based on architecture? If its a command line app.

**Comment by ghost** [comment]
Yes, but we do need to really parse and understands the sln/props/vcxpoj to do so and have a real added value. 

This is absolutely not the case for now. It should really be interesting, to do it (and would provide way more features) but I think this is a load of work compared to the usage. 

Wouldn't a simple shell exec be enough for most of the cases ?

https://packagecontrol.io/packages/Shell%20Exec
